BookingsAPI/Bookings.DAL/Commands/RemoveHearingCommand.cs

```diff
@@ -29,14 +31,26 @@ public async Task Handle(RemoveHearingCommand command)
         {
             var hearing = await _context.VideoHearings
                 .SingleOrDefaultAsync(x => x.Id == command.HearingId);
-            
+
             if (hearing == null)
             {
                 throw new HearingNotFoundException(command.HearingId);
             }
 
+            hearing.Participants.ToList().ForEach(x => RemoveQuestionnaires(x.Questionnaire));
+            await _context.SaveChangesAsync();
```

Comment:
I think we would be alright with just one save which would also make it more transactional, not sure if it's entirely transactional though

Reply:
It's changed